backend/Dockerfile.dev

```diff
@@ -0,0 +1,21 @@
+FROM pennlabs/django-base:9a59b9133a4c82d8391b282a3a39685d7ac6738d
+
+LABEL maintainer="Penn Labs"
+
+# Copy project dependencies
+COPY Pipfile* /app/
+
+# Install project dependencies
+RUN pipenv install --system
+
+# Copy project files
+COPY . /app/
+
+ENV DJANGO_SETTINGS_MODULE Platform.settings.production
+ENV SECRET_KEY 'temporary key just to build the docker image'
+
```

Comment:
i know nothing about devops but don't you also need a `ENV DEV_LOGIN True` or something because Platform.settings.production sets the variable `IS_DEV_LOGIN` by searching for an environment var DEV_LOGIN and defaulting to False.

Reply:
We can talk more about it if you're interested in learning, but the gist is that the Dockerfile just builds an "image" which is essentially a snapshot of platform that can be run using docker. That means the Dockerfile itself shouldn't have specific configuration tied to how we deploy platform-dev. If you take a look at the changes in k8s/values.yaml, there's a section setting DEV_LOGIN to True there. That way when we run the docker image in Kubernetes, it'll correctly turn on the dev login feature